appcues/src/main/java/com/appcues/ui/primitive/TextInputPrimitive.kt

```diff
@@ -77,10 +74,9 @@ internal fun TextInputPrimitive.Compose(modifier: Modifier) {
 
         // Several styling customization options for TextField noted here https://stackoverflow.com/a/68592613
         TextField(
-            value = text.value,
+            value = formState.getValue(this@Compose).value,
```

Comment:
where we now just use the mutableState from the form state, rather than having a shadow copy in a separate `text` variable like we had before.